### PR TITLE
Enabling building for ARM in dnx native parts

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: build/Config.Definitions.Props [hunk 1]
```diff
@@ -17,5 +21,9 @@
       <Configuration>Release</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|ARM">
+      <Configuration>Debug</Configuration>
```
Comment (BrennanConroy) [inline build/Config.Definitions.Props]: Release?

Comment (moozzyk) [inline build/Config.Definitions.Props]: Yes. Will fix.

File: src/dnx/dnx.vcxproj [hunk 2]
```diff
@@ -15,6 +15,12 @@
     <ProjectName Condition="'$(BuildForOneCore)' == 'True'">dnx.onecore</ProjectName>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <Defines Condition="$(RuntimeType) == 'CORECLR_WIN'">CORECLR_WIN</Defines>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Platform)' == 'ARM'">
+    <ProjectName>dnx</ProjectName>
+    <ConfigurationType>Application</ConfigurationType>
+  </PropertyGroup>
+  <PropertyGroup>
     <OutDir>bin\$(Platform)\$(Configuration)\$(TargetFramework)\</OutDir>
```
Comment (BrennanConroy) [inline src/dnx/dnx.vcxproj]: These were in the "Globals" PropertyGroup, is it okay that they aren't anymore?

Comment (moozzyk) [inline src/dnx/dnx.vcxproj]: "Globals" is just a label - it does not really matter. I will add them just for completeness.

**Comment by moozzyk** [comment]
:watch:

**Comment by davidfowl** [comment]
Why is everything failing ci wise?

**Comment by moozzyk** [comment]
AppVeyor "build cancelled" - I have not seen AppVeyor build passing in ages. 
Travis builds failed due to failed restore: 
Error: FindPackagesById: System.Resources.ReaderWriter
  The Task was canceled

**Comment by moozzyk** [comment]
Also note it's mostly a no-op because we currently don't build ARM on the CI

**Pull request closed without merging**